packages/Material-Core.package/MDRenderer.class/class/roundRectShapeBounds.radius.borderWidth..st

```diff
@@ -1,75 +1,75 @@
 drawing
```

Comment:
file too long. Since it is quite hard to shorten it, this comment is only a suggestion